spec/temporal.html

```diff
@@ -194,7 +194,7 @@ <h1>SystemUTCEpochNanoseconds ( )</h1>
       <emu-alg>
         1. Let _ns_ be the approximate current UTC date and time, in nanoseconds since the epoch.
         1. [id="step-temporal-systemutcepochnanoseconds-clamp"] Set _ns_ to the result of clamping _ns_ between −8.64 × 10<sup>21</sup> and 8.64 × 10<sup>21</sup>.
-        1. Return _ns_.
+        1. Return ℤ(_ns_).
```

Comment:
As a follow up, we should assert in CreateTemporalZonedDateTime that the epoch-ns value is a BigInt.